addons/im_livechat/controllers/chatbot.py

```diff
@@ -63,22 +62,34 @@ def chatbot_trigger_step(self, channel_id, chatbot_script_id=None):
             return None
 
         posted_message = next_step._process_step(discuss_channel)
-        return {
-            "data": Store(posted_message, for_current_user=True).get_result(),
-            'scriptStep': {
-                'id': next_step.id,
-                'answers': [{
-                    'id': answer.id,
-                    'label': answer.name,
-                    'redirectLink': answer.redirect_link,
-                } for answer in next_step.answer_ids],
-                'isLast': next_step._is_last_step(discuss_channel),
-                'message': plaintext2html(next_step.message) if not is_html_empty(next_step.message) else False,
-                'type': next_step.step_type,
+        store = Store()
+        store.add(posted_message, for_current_user=True)
+        store.add(next_step)
+        store.add(
+            "ChatbotStep",
+            {
+                "id": (next_step.id, discuss_channel.id),
+                "isLast": next_step._is_last_step(discuss_channel),
+                "message": store.one(posted_message, only_id=True),
```

Comment:
`one` can be used from the class rather than instance `Store.one`

Reply:
Any reason to do so? I would have said: use instance if available, class otherwise.

Reply:
It gives wrong impression it depends on current `store`?

Also `Store.one` is easy to find, whereas the instance is more likely to be renamed, that makes refactoring a little harder.